tests/lib/tester.js

```diff
@@ -29,6 +29,11 @@ function getRaw(ast) {
             return undefined; // eslint-disable-line no-undefined
         }
 
+        // JSON cannot handle BigInt.
+        if (typeof value === "bigint") {
+            return null;
```

Comment:
The tests depend on JSON heavily, but JSON doesn't support BigInt values. So tests handle `Literal#value` as null. I think that this is not a big problem because it's null if runtime doesn't support bigint natively.